site/ecosystem.md

```diff
@@ -41,6 +41,7 @@ If you have a project you'd like to include in this list, either [click here](ht
 * [fs2-rabbit](https://fs2-rabbit.profunktor.dev/): RabbitMQ stream-based client built on top of fs2 and cats effect.
 * [fs2-reactive-streams](https://github.com/zainab-ali/fs2-reactive-streams): A reactive streams implementation for fs2.
 * [fs2-ssh](https://github.com/slamdata/fs2-ssh): A wrapper around Apache SSHD targeting cats-effect and fs2.
+* [fs2-trottler](https://github.com/kovstas/fs2-throttler): Throttling for fs2 based on the token bucket algorithm.
```

Comment:
Thanks! That should be `throttler`, right?

Reply:
Yes, thanks!